keyboards/ekow/akira/keymaps/via/rules.mk

```diff
@@ -0,0 +1 @@
+VIA_ENABLE = yes
```

Comment:
Still need to add a `keymap.c` file to make this a valid keymap, or remove this file.

Reply:
I add a keymap.c file in the via directory, thank you  again!